qcp/gates.py

```diff
@@ -202,6 +199,8 @@ def _generic_control(size: int, controls: List[int],
     target_bit = 2 ** target
 
     for i in range(0, n):
+        condition = (i & target_bit) == target_bit and i & mask >= mask
```

Comment:
This looks like a line that didn't get removed after a merge from master, L204 & L205 cover those checks already.

Reply:
yeah i probably missed that